bumble/gatt.py

```diff
@@ -271,6 +271,26 @@ def __str__(self):
         return f'Characteristic(handle=0x{self.handle:04X}, end=0x{self.end_group_handle:04X}, uuid={self.uuid}, properties={Characteristic.properties_as_string(self.properties)})'
 
 
+# -----------------------------------------------------------------------------
+class CharacteristicDeclaration(Attribute):
```

Comment:
this class doesn't seem to be used here. Is the intent that a client would use that class directly for its own purposes?

Reply:
sorry, forgot to git-add that section of the PR. see newest revision